lib/Swagger/src/Discussion/Models/UpvoteHalResponse.php

```diff
@@ -66,7 +66,7 @@ class UpvoteHalResponse implements ArrayAccess
         'upvote_count' => 'int',
         '_links' => '\Swagger\Client\Discussion\Models\HalLinks',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "_links" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)